UI/src/model/selectedMassTransitStop.js

```diff
@@ -663,7 +665,8 @@
       isSuggested: isSuggested,
       isAnAddToolOption: isAnAddToolOption,
       isTerminalType: isTerminalType,
-      isServicePointType: isServicePointType
+      isServicePointType: isServicePointType,
+      getServicePointPropertyOrdering: servicePointPropertyOrdering
```

Comment:
Keep the same idea already existent on the other methods.
```javascript
getServicePointPropertyOrdering: getServicePointPropertyOrdering

Reply:
Applied